libs/sdk-ui-kit/src/Dropdown/DropdownTabs.tsx

```diff
@@ -11,6 +11,24 @@ export interface IDropdownTagsProps {
     onTabSelect?: (tab: ITab) => void;
 }
 
+/**
+ * @internal
+ */
+export const INSIGHT_DROPDOWN_MY_TAB_ID = "gs.visualizationsList.tabs.my";
```

Comment:
Should these be in here? Maybe AD/KD would be a better place for this, what do you think? Even at the "cost" of duplication.

Reply:
No problem, I will duplicate them (as they these constants are used in both AD and KD).